platform/javascript/package-lock.json

```diff
@@ -0,0 +1,1605 @@
+{
+  "name": "godot",
+  "version": "1.0.0",
+  "lockfileVersion": 1,
```

Comment:
I'm not very familiar with the JS ecosystem, but is it really useful to include the lock file in Git (at least for style tooling and not something actually used by the engine code)? Otherwise we could add it to `.gitignore` and save 1600 lines of node modules that is likely going to change quite often.

Reply:
Instead, if we don't want the code style to change under our nose while running on CI, we can hard-pin a specific version in the `package.json` and bump it manually.

Reply:
Yeah, npm sucks, that file is supposed to be added to the version control:
https://github.com/npm/cli/blob/release-6.14.7/docs/content/configuring-npm/package-lock-json.md